avb/Android.mk

```diff
@@ -55,6 +55,10 @@ ifneq ($(KERNELFLINGER_DISABLE_DEBUG_PRINT),true)
     LOCAL_CFLAGS += -DAVB_ENABLE_DEBUG
 endif
 
+ifeq ($(BOARD_AVB_ENABLE),true)
+    LOCAL_CFLAGS += -DAVB_AB_I_UNDERSTAND_LIBAVB_AB_IS_DEPRECATED
```

Comment:
"AVB_AB_I_UNDERSTAND_LIBAVB_AB_IS_DEPRECATED" this macro name is bit strange...

Reply:
AVB required.